receiver/netflowreceiver/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+type: netflow
+scope_name: otelcol/netflowreceiver
+
+status:
+  class: receiver
+  stability:
+    development: [logs]
+  distributions:
+    - contrib
```

Comment:
I don't think we should include `contrib` here until it's added to the `contrib` release. If we advertise that this receiver is included in `contrib`, users will be confused when it's not working as the README states. It's caused confusion for users a couple times in the past.